<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>homefile</class>
 <widget class="QMainWindow" name="homefile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>621</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>HOMEPAGE</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-9</x>
      <y>-9</y>
      <width>831</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 255);
color: rgb(0, 255, 255);
font: 20pt &quot;MV Boli&quot;;</string>
    </property>
    <property name="lineWidth">
     <number>13</number>
    </property>
    <property name="text">
     <string>Анализатор транзакций</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="name_check">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 0, 255);
font: 12pt &quot;MV Boli&quot;;</string>
       </property>
       <property name="placeholderText">
        <string>Имя:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="password_check">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 0, 255);
font: 12pt &quot;MV Boli&quot;;</string>
       </property>
       <property name="placeholderText">
        <string>Пароль:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>440</y>
      <width>318</width>
      <height>87</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="enter_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
color: rgb(0, 255, 255);
background-color: rgb(200, 0, 235);
font: 14pt &quot;MS Shell Dlg 2&quot;;
border: 1px solid rgb(200, 0, 245);
border-radius: 7px;
width: 100 px;
height: 45px;
}
QPushButton:hover {
background-color: rgb(255, 129, 245);
}
QPushButton:pressed {
background-color: rgba(200, 0, 245, 65);
}</string>
       </property>
       <property name="text">
        <string>Войти</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="signup_btn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
color: rgb(0, 255, 255);
background-color: rgb(200, 0, 235);
font: 14pt &quot;MS Shell Dlg 2&quot;;
border: 1px solid rgb(200, 0, 245);
border-radius: 7px;
width: 100 px;
height: 45px;
}
QPushButton:hover {
background-color: rgb(255, 129, 245);
}
QPushButton:pressed {
background-color: rgba(200, 0, 245, 65);
}</string>
       </property>
       <property name="text">
        <string>Регистрация</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>176</y>
      <width>671</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>28</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 255);
font: 75 28pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 255);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:28pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:400;&quot;&gt;Этот анализатор используется для контроля осуществляемых доходов и расходов. Он подсчитывает общую сумму затраченных и полученных денег, определяет сферы транзакций. В данном анализаторе можно вввести на что были затрачены деньги(или получены), кратко описать и указать дату совершенной транзакции. Также можно удалить определенную транзакцию, предварительно выбрав её.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>320</y>
      <width>281</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>assets/p_home.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
